### PR TITLE
fix: css :not() rules with multi-selectors broken in AMP

### PR DESCRIPTION
**Pull request by dkoo**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-theme/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes several instances of a CSS selector that AMP doesn't like:

```css
.selector:not( .sub-selector-1, .sub-selector-2 )
```

should be:

```css
.selector:not( .sub-selector-1 ):not( .sub-selector-2 )
```

These selectors tend to target highly specific groups of blocks that mainly appear on homepages.

### How to test the changes in this Pull Request:

1. On a fresh test site, enable AMP and install the [bad theme version](https://github.com/Automattic/newspack-theme/releases/tag/v1.69.1).
2. View the homepage and observe that it's broken, mainly because all the CSS below the first instance of the `:not` combo-selector is not compiled by AMP.

Screen Shot 2023-03-14 at 4 38 05 PM

3. Check out this branch, rebuild CSS, confirm that the homepage renders as expected.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: newspack-theme/sass/plugins/newspack-ads.scss [hunk 1]
```diff
@@ -125,20 +125,6 @@ body {
 			z-index: 1;
 		}
 	}
-
-	&.single-featured-image-behind:not(.newspack-front-page, .style-pack-style-2)
```
Comment (laurelfulford) [inline newspack-theme/sass/plugins/newspack-ads.scss:129]: Sorry, this was the result of my bad explanation! It's specifically the `.style-pack-style-2` CSS class that doesn't exist anymore -- we still need these styles for two really specific use cases (it makes sure these ad styles aren't applied when the simplified subpage header is paired with either the featured image behind or featured image beside on posts and pages). 

"Style pack 2" eventually became Newspack Nelson but it doesn't look like the styles are still needed for that (or at least I can't see any issues comparing Newspack Nelson with these settings to other themes on master -- I'm betting it was a specificity thing!).

File: newspack-theme/sass/plugins/newspack-ads.scss [hunk 2]
```diff
@@ -125,20 +125,6 @@ body {
...
-		.global_below_header
-		+ .site-content {
-		margin-top: 0;
-	}
-
-	@include utilities.media( tablet ) {
-		&.single-featured-image-beside:not(.newspack-front-page, .style-pack-style-2)
```
Comment (laurelfulford) [inline newspack-theme/sass/plugins/newspack-ads.scss:136]: See above! Only the `.style-pack-style-2` bit isn't needed, the rest of these styles look like they are still needed!

**Review by laurelfulford — CHANGES_REQUESTED** [review]
I found one small error that was the result of my poor explanation around styles that weren't needed --- I added code comment but let me know if I'm not making sense!

Otherwise, this looks good to me! 🙂

File: newspack-theme/sass/plugins/newspack-ads.scss (hunk 1 above)
Comment (dkoo) [reply newspack-theme/sass/plugins/newspack-ads.scss:129]: 397fd68 restores these styles without the `.style-pack-style-2` selector.

File: newspack-theme/sass/plugins/newspack-ads.scss (hunk 2 above)
Comment (dkoo) [reply newspack-theme/sass/plugins/newspack-ads.scss:136]: 397fd68 restores these styles without the `.style-pack-style-2` selector.

**Review by laurelfulford — APPROVED** [review]
🙌 Thanks @dkoo! This looks good to me!

**Comment by dkoo** [comment]
As expected, Stylelint doesn't like the AMP way of doing these selectors. We should update the config in `newspack-scripts` to address.

**Pull request merged**